renderers/photoaudiohaptics.schema.json

```diff
@@ -0,0 +1,91 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://image.a11y.mcgill.ca/renderers/photoaudiohaptics.schema.json",
+    "title": "Photo Audio Haptic Renderer",
+    "type": "object",
+    "description": "Data for basic audio haptics selection renderer ca.mcgill.a11y.image.renderer.PhotoAudioHaptics.",
+    "definitions": {
+        "centroid": {
+            "type": "array",
+            "items": {
+                "type": "number",
+                "minimum": 0,
+                "maximum": 1
+            },
+            "minItems": 2,
+            "maxItems": 2
+        },
+        "coordinates": {
+            "type": "array",
+            "items": {
+                "type": "number",
+                "minimum": 0,
+                "maximum": 1
+            }
+        },
+        "contourPoints": {
+            "type": "array",
+            "items": {
+                "allOf": [
+                    {
+                        "$ref": "#/definitions/coordinates"
+                    }
+                ]
+            }
+        }
+    },
+    "properties": {
+        "image": {
```

Comment:
I don't think you need this? I'm loathe to include the graphic (deprecating "image" use see #224), but if you do really need it then there needs to be more clarification as to format than "string".

Reply:
Tagging @Cybernide  for input as well but based on some previous convos having the image in the haptic rendering would be useful for user testing.

Reply:
Not saying don't include the image, just that there should really be a pipeline to accessing it in the extension...it doesn't make sense to me for the extension to send a blob and then the server to return that same blob unmodified.

Reply:
Ah yes, Sri, we should be referring to them as _graphics_ from here on in to avoid confusion. In fact, do read #224, which which involves refactoring all references from "images" to "graphics".

But I do agree that it would be good to have a copy of the graphic in the rendering "area?" it would help partially sighted users. Can we make a work item to make this pipeline?

Reply:
Makes sense. I think we can probably pass the blob directly into the pop-up menu from the browser.